crates/primitives/Cargo.toml

```diff
@@ -96,10 +96,11 @@ arbitrary = [
     "alloy-eips/arbitrary",
     "dep:arbitrary",
     "dep:proptest",
-    "zstd-codec",
+    "zstd"
 ]
 c-kzg = ["dep:c-kzg", "revm-primitives/c-kzg", "dep:tempfile", "alloy-eips/kzg"]
-zstd-codec = ["dep:zstd"]
+zstd = ["dep:zstd"]
+no-zstd = []
```

Comment:
I find these a bit confusing because not obvious why `no-std` isn't the absence of `zstd`